Initial webpage

amittcohen opened the pull request:
Creating a webpage that contains:
* Project's name
* Project's description
* Mentors and students names
* Design of the webpage

Yarboa reviewed (changes requested):
Is it ready for submission? I see empty page 
> Find a name for the project
> Write a decription for the project
> Design of the webpage

Is it the django landing page?

amittcohen commented:
> Is it ready for submission? I see empty page
> 
> > Find a name for the project
> > Write a decription for the project
> > Design of the webpage
> 
> Is it the django landing page?

Not ready. just an initial one to update

Yarboa commented:
Please Mark it as draft,
Please refer this link
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request

ShirleyFichman reviewed (approved):
LGTM

taljacob2 commented on webpage.html:
```diff
@@ -0,0 +1,27 @@
+<head>
+<title>Job Seeker</title>
+<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Can you please download the CSS files, place them in a folder and refer to them, instead of using CDN?
- Because it might happen that those CDN links would be removed or break sometime in the future.
- Also, it will allow us to render the page while not concerning of being connected to the internet.

amittcohen replied:
Downloaded the css file

taljacob2 reviewed (changes requested):
- This page looks great!
  Can you run it through our Django application?

- Can you please add some content to this PR's description?
  Explain shortly the feature you have added and what it contains.

- Also, I suggest you `rebase fixup` all of the commits on this PR into a single commit, since each commit here is minor, so they all could be considered as one.

taljacob2 reviewed (approved):
Good Job! 🥇 
Our next mission is to run it as the `index.html` page of Django.

Let's do it in another PR.
(Or maybe we should rebase this PR on the `main` and continue contributing on this PR? @Yarboa @kasemAlem what do you think is the correct way?) 

 I can make an issue for guidance.

taljacob2 commented:
> Good Job! 🥇 Our next mission is to run it as the `index.html` page of Django.
> 
> Let's do it in another PR. (Or maybe we should rebase this PR on the `main` and continue contributing on this PR? @Yarboa @kasemAlem what do you think is the correct way?)
> 
> I can make an issue for guidance.

@amittcohen I made a new issue for you at #15

Yarboa reviewed (approved):
We will avoid direct use of CSS in the projects
We recommend to use boostrap lib, css in included, 

Removed approval, no django redirection to this home page

Yarboa reviewed (changes requested):
See previous comment, need to set it as default home page in django

amittcohen commented on jobseeker/templates/jobseeker/index.html:
```diff
@@ -0,0 +1,25 @@
+      <h1>Project Description</h1>
+      <h5 class="w3-padding-32">Job Seeker is a job board website that is used both by job-seekers and employers:<br>Job-seekers can use Job Seeker to search for new job opportunities in their area and profession.<br>Employers can use Job Seeker to advertise their open positions and find new candidates.</h5>
+    </div>
+    <div class="w3-third w3-center">
```
Tried to add the logo here but it failed for some reason..

paOmer replied:
Your'e gonna need to make a few changes in order to load the logo file:

1. add the following img tag inside the div at line 20 (if that where you wish to place it) `<img src="{% static 'jobseeker/logo.png' %}" />`
2. Add the following line to the top of this HTML page: `{% load static %}`
3. Add the following directories to the jobseeker app directory: `static/jobseeker/` and place the logo file there.
4. Add the following urls.py file in the jobseeker app directory:
  ```
from django.urls import path
  from . import views
  
  urlpatterns = [
      path('', views.index, name='index'),
  ]
```
5. Change the app urls.py file to look like the following:
  ```
from django.contrib import admin
  from django.urls import path, include
  
  urlpatterns = [
      path('admin/', admin.site.urls),
      path('', include('jobseeker.urls')),
  ]
```

amittcohen replied:
Works! thanks a lot

amittcohen commented:
In the last commits:
- Added the webpage to the django application following the instruction in part 5 of the tool introduction challenge.
- Removed the downloaded css files and put a link instead as requested.

DeanBiton reviewed (approved):
LGTM

Yarboa reviewed (approved):
LGTM

taljacob2 reviewed (approved):
LGTM 🥇

taljacob2 commented on webpage.html:
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```
@amittcohen We may remove this file entirely.
